ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/InitializeParallelIngestion.scala

```diff
@@ -55,15 +55,11 @@ private[platform] case class InitializeParallelIngestion(
       _ <- dbDispatcher.executeSql(metrics.daml.parallelIndexer.initialization)(
         ingestionStorageBackend.deletePartiallyIngestedData(ledgerEnd)
       )
-      _ <- ledgerEnd match {
-        case Some(ledgerEnd) => updatingStringInterningView.update(ledgerEnd.lastStringInterningId)
-        case None => Future.unit
-      }
-      ledgerEndOrBeforeBegin = ledgerEnd.getOrElse(ParameterStorageBackend.LedgerEndBeforeBegin)
+      _ <- updatingStringInterningView.update(ledgerEnd.lastStringInterningId)
```

Comment:
I did not keep the original optimization of not running the `update` if there is no data in the ledger because it makes the code simpler - the `update` method already handles the case of an empty id range.